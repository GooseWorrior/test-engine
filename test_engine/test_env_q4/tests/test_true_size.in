newDefault 1
size 1
[]= 1 5 5
size 1
[]= 1 3 3
size 1
[]= 1 4 4
size 1
[]= 1 2 2 
size 1
[]= 1 1 1
size 1
[]= 1 7 7 
size 1
[]= 1 6 6
size 1
[]= 1 8 8
size 1
[]= 1 9 9
size 1
erase 1 1
size 1
erase 1 2
size 1
erase 1 3
size 1
erase 1 4
size 1
erase 1 5 
size 1
erase 1 6
size 1
erase 1 7
size 1
erase 1 8
size 1
erase 1 9
size 1
newInit 2
size 2
erase 2 1
size 2
