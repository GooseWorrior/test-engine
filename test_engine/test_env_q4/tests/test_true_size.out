1:
0
1:
1: (5,5)
1
1: (5,5)
1: (3,3) (5,5)
2
1: (3,3) (5,5)
1: (3,3) (4,4) (5,5)
3
1: (3,3) (4,4) (5,5)
1: (2,2) (3,3) (4,4) (5,5)
4
1: (2,2) (3,3) (4,4) (5,5)
1: (1,1) (2,2) (3,3) (4,4) (5,5)
5
1: (1,1) (2,2) (3,3) (4,4) (5,5)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (7,7)
6
1: (1,1) (2,2) (3,3) (4,4) (5,5) (7,7)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7)
7
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8)
8
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
9
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1: (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
8
1: (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1: (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
7
1: (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1: (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
6
1: (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1: (5,5) (6,6) (7,7) (8,8) (9,9)
5
1: (5,5) (6,6) (7,7) (8,8) (9,9)
1: (6,6) (7,7) (8,8) (9,9)
4
1: (6,6) (7,7) (8,8) (9,9)
1: (7,7) (8,8) (9,9)
3
1: (7,7) (8,8) (9,9)
1: (8,8) (9,9)
2
1: (8,8) (9,9)
1: (9,9)
1
1: (9,9)
1:
0
1:
1:
2: (1,2) (3,4) (5,6) (7,8)
4
1:
2: (1,2) (3,4) (5,6) (7,8)
1:
2: (3,4) (5,6) (7,8)
3
1:
2: (3,4) (5,6) (7,8)
